README.md

```diff
@@ -32,3 +28,152 @@
   1. Socket.io and Redis
   2. Popular modules
   3. Links
+
+## Introduction to NodeJS ##
```

Comment:
In this section, you will learn the basics of NodeJS. From how NodeJS application organize their source code, to how application dependencies are handled by NPM, and which global objects are provided by default.